app/views/blog_posts/index.html.erb

```diff
@@ -14,6 +14,8 @@
         <% end %>
       </div>
     <% end %>
+    <% scope = BlogPost.all.page(params[:page]) %>
+    <%= paginate(scope) %>
```

Comment:
You should use the instance variable here (`@blog_posts`) since that's what you want to paginate.